.github/workflows/build.yml

```diff
@@ -355,22 +353,15 @@ jobs:
         with:
           maven-version: ${{ env.MAVEN_VERSION }}
 
-      - name: Download liquibase-commercial
-        uses: liquibase/action-download-artifact@v2-liquibase
-        with:
-          github_token: ${{ secrets.BOT_TOKEN }}
-          workflow: build.yml
-          workflow_conclusion: success
-          branch: "${{ needs.setup.outputs.proBranchName }}"
-          name: liquibase-commercial-modules
-          path: download/liquibase-commercial
-          repo: liquibase/liquibase-pro
+      # Download liquibase-commercial from GPM
 
       - name: Install Built Modules
         run: |
           (find . -name *-SNAPSHOT.jar -exec mvn -B org.apache.maven.plugins:maven-install-plugin:3.0.0-M1:install-file -Dfile={} \;)
           (find . -name *-SNAPSHOT-sources.jar -exec mvn -B org.apache.maven.plugins:maven-install-plugin:3.0.0-M1:install-file -Dclassifier=sources -Dfile={} \;)
-          mvn -B org.apache.maven.plugins:maven-install-plugin:3.0.0-M1:install-file -Dfile=download/liquibase-commercial/liquibase-commercial-0-SNAPSHOT.jar
+          mvn org.apache.maven.plugins:maven-dependency-plugin:3.5.0:get -Dartifact=org.liquibase:liquibase-commercial:0-SNAPSHOT -DrepoUrl=https://maven.pkg.github.com/liquibase/liquibase-pro
```

Comment:
Version 0-SNAPSHOT shouldn't be on GPM to download this way? And since it is available from GPM which maven understands how to auto-download from you shouldn't have to manually get and install it at all.

So I think this line plus the next 2 are not needed at al anymore? Instead have a `liquibase-commercial.version` property which can be set in any of the maven calls that will be using liquibase-commercial?